examples/event-webhooks-local-flask/README.md

```diff
@@ -0,0 +1,278 @@
+# Benchling App Example: Entity Registered Event Webhook Data Sync
+
+An example Benchling App written in Python which demonstrates receiving a `v2.entity.registered`
+event from Benchling via a webhook subscription. The data is retrieved from the API and written
+to an external database.
+
+![image info](./docs/demo-full.gif)
+
+> ℹ️ Events delivered by webhooks is in early access and unavailable by default. To use this example, please reach out to Benchling support to enable webhook delivery for events on your tenant.
```

Comment:
@Psterne @dgervais-benchling I didn't see any public docs on getting setup with Events over Webhooks. I also used "Events delivered by webhooks" as I feel like "events over webhooks" is our internal language and not necessarily intuitive to users.

Please suggest any alternative copy you'd like here or elsewhere in the `README`!

Reply:
@bowenwr Updated docs for Events (via Webhooks and Eventbridge) are WIP. We can link to it from here once those are live. 

I agree that "Events delivered by webhooks" is better. At some point I'm hoping we can just call it "Webhooks"!